pjsip/src/pjsip/sip_transport.c

```diff
@@ -2157,20 +2157,43 @@ PJ_DEF(pj_ssize_t) pjsip_tpmgr_receive_packet( pjsip_tpmgr *mgr,
             msg_status = pjsip_find_msg(current_pkt, remaining_len, PJ_FALSE, 
                                         &msg_fragment_size);
             if (msg_status != PJ_SUCCESS) {
-                if (remaining_len == PJSIP_MAX_PKT_LEN) {
-                    mgr->on_rx_msg(mgr->endpt, PJSIP_ERXOVERFLOW, rdata);
-                    
-                    /* Notify application about the message overflow */
+                pj_status_t dd_status = msg_status;
+
+                if (msg_status == PJSIP_EPARTIALMSG) {
+                    if (remaining_len != PJSIP_MAX_PKT_LEN) {
+                        /* Not enough data in packet. */
+                        return total_processed;
+                    }
+                    dd_status = PJSIP_ERXOVERFLOW;
```

Comment:
- Better put comment for future ref about the two cases of `EPARTIALMSG`, i.e: really partial message or overflow (buffer too small or packet too large)?
- And/or print log (as in the other errors)?